tox.ini

```diff
@@ -40,14 +40,14 @@ commands =
     format: isort --check .
     format: black -l 80 --include \.pct\.py$ --check notebooks
     # tests (run with "-- --runslow yes" to run all tests)
-    tests: pip install . -r tests/requirements.txt -c tests/constraints.txt
+    tests: pip install .[qhsri] -r tests/requirements.txt -c tests/constraints.txt
     tests: pytest --ignore trieste/experimental {posargs}
     # docs build
-    docs: pip install . -r notebooks/requirements.txt -c notebooks/constraints.txt
+    docs: pip install .[qhsri] -r notebooks/requirements.txt -c notebooks/constraints.txt
     docs: pip install -r docs/requirements.txt -c docs/constraints.txt
     docs: bash -c "cd docs; make html"
     # quickdocs build
-    quickdocs: pip install . -r notebooks/requirements.txt -c notebooks/constraints.txt
+    quickdocs: pip install .[qhsri] -r notebooks/requirements.txt -c notebooks/constraints.txt
```

Comment:
You'll also need to update `tests_latest`.

Though I'm now concerned that if we always test with `[qhsri]` then we won't spot a dependency break in the basic trieste package. So maybe you should _only_ test qhsri with `tests_latest` (and `docs` and `quickdocs`), and have the normal `tests` just not use qhsri and instead skip the qhsri tests?